test/ng/directive/ngOptionsSpec.js

```diff
@@ -2641,6 +2658,23 @@ describe('ngOptions', function() {
       dealoc(element);
     }));
 
+
+    it('should ignore the comments within the select', function() {
```

Comment:
There are no expectations regarding the comments you added. Should this test  that all comments that are not part of the empty option are removed?